src/components/Login.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Form from "react-validation/build/form";
+import Input from "react-validation/build/input";
+import CheckButton from "react-validation/build/button";
+import { login } from "../actions/auth";
+
+const required = (value) => {
+    if (!value) {
+        return (
+            <div className="alert alert-danger" role="alert">
+                This field is required!
+            </div>
+        );
+    }
+};
+const Login = (props) => {
+    const form = useRef();
+    const checkBtn = useRef();
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [loading, setLoading] = useState(false);
+    const { isLoggedIn } = useSelector(state => state.auth);
```

Comment:
isLoggedIn should get from local storage because if you reload the page, you would lose data

Reply:
Removed. Checking from local storage now.